Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Payment intent.
+///
+/// - Note: Must be set to BTPayPalRequestIntentSale for immediate payment, BTPayPalRequestIntentAuthorize to authorize a payment for capture later, or BTPayPalRequestIntentOrder to create an order. Defaults to BTPayPalRequestIntentAuthorize. Only applies to PayPal Checkout.
+///
+/// [Capture payments later reference](https://developer.paypal.com/docs/integration/direct/payments/capture-payment/)
+///
+///[Create and process orders reference](https://developer.paypal.com/docs/integration/direct/payments/create-process-order/)
+@objc public enum BTPayPalRequestIntent: Int {
+    /// Authorize
+    case authorize
+
+    /// Sale
+    case sale
+
+    /// Order
+    case order
+}
+
+///  The call-to-action in the PayPal Checkout flow.
+///
+///  - Note: By default the final button will show the localized word for "Continue" and implies that the final amount billed is not yet known.
+///  Setting the BTPayPalRequest's userAction to `BTPayPalRequestUserActionCommit` changes the button text to "Pay Now", conveying to
+///  the user that billing will take place immediately.
+@objc public enum BTPayPalRequestUserAction: Int {
+    /// Default
+    case `default`
```

Comment:
```suggestion
    case default
```

did we need this backticks?

Reply:
Yeah we ned the backticks here if we want to use `default`. It's a [protected identifier and needs to be escaped](https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID413).

Reply:
Since this change is happening in the v6beta, we can make breaking changes if we want, right?

Maybe us needing to use the `` is a sign we could come up with a better name for this enum case option.

Maybe `.continue` and `.commit`, or `.continue` and `.payNow` even?

Reply:
Yeah! We can certainly make this a breaking change. Agree it's quite odd - on NXO we used `case none` for optional obj-c exposed enum defaults ([example here](https://github.paypal.com/nativexo/ios-native-checkout/blob/db68545f03ba400e23d21e4ece64b63a1ba09d92/Sources/PayPalCheckoutIO/Models/OrderApplicationContext.swift#L97)). I think that worked well and was clear. So we could do `.payNow` and `.none` as the case options here and switch the other use of default to none as well.

Reply:
Or like you said if we want continue to be more explicit `. continue` and `.payNow` works even better!

Reply:
Updated to what I think makes the most sense here with avoiding backticks but let me know your thoughts: https://github.com/braintree/braintree_ios/pull/912/commits/bd7439fa850ee1c74c1d76e84871a0d83f93d2ea